code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -94,6 +94,7 @@
 
 /obj/machinery/chem_dispenser/Initialize()
 	. = ..()
+	message_admins("A reagent dispensor at [ADMIN_VERBOSEJMP(src)]")
```

Comment:
NICE FUCKING INITIALIZE SPAM
USE MAPLOAD.

Reply:
> NICE FUCKING INITIALIZE SPAM USE MAPLOAD.

shut the fuck up and go back to coding bob

Reply:
He do be right tho

Reply:
Are you happy?